.github/workflows/agp-matrix.yml

```diff
@@ -4,25 +4,20 @@ on:
   push:
     branches:
       - main
-      - release/**
   pull_request:
 
-jobs:
-  cancel-previous-workflow:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Cancel Previous Runs
-        uses: styfle/cancel-workflow-action@01ce38bf961b4e243a6342cbade0dbc8ba3f0432 # pin@0.12.0
-        with:
-          access_token: ${{ github.token }}
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
 
+jobs:
   agp-matrix-compatibility:
     timeout-minutes: 30
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        agp: [ '8.0.0','8.1.0-alpha11' ]
+        agp: [ '8.0.0','8.1.4','8.2.0','8.3.0-beta01' ]
```

Comment:
May we skip some intermediate versions, like the `8.1.4` and perhaps `8.2.0`?

Reply:
yeah, I thought about that, but AGP sometimes introduces a change, and then fixes it in the following version, so I think it's good if we are aware if it's broken in e.g. `8.1.4`,  but fixed in `8.2.0`, it will save us time investigating issues. Also, they run in parallel, so shouldn't be noticable, but I would keep them at 4-5 max.